src/test/resources/org/rust/ide/intentions/fixtures/unwrap_lambda_expr/unwrap_braces.rs

```diff
@@ -0,0 +1,3 @@
+fn main () {
+    |x| { x *<caret> x }
```

Comment:
ok, when nested blocks it will indeed encounter a problem, i will refer the Kotlin implementation.